src/host/buildvm_asm.c

```diff
@@ -39,6 +39,34 @@ static void emit_asm_reloc(BuildCtx *ctx, int type, const char *sym) {
   }
 }
 
+static const char *const jccnames[] = {
+  "jo", "jno", "jb", "jnb", "jz", "jnz", "jbe", "ja",
+  "js", "jns", "jpe", "jpo", "jl", "jge", "jle", "jg"
+};
+
+/* Emit relocation for the incredibly stupid OSX assembler. */
+static void emit_asm_reloc_mach(BuildCtx *ctx, uint8_t *cp, int n,
+                                const char *sym)
+{
+  const char *opname = NULL;
+  if (--n < 0) goto err;
+  if (cp[n] == 0xe8) {
+    opname = "call";
+  } else if (cp[n] == 0xe9) {
+    opname = "jmp";
+  } else if (cp[n] >= 0x80 && cp[n] <= 0x8f && n > 0 && cp[n-1] == 0x0f) {
+    opname = jccnames[cp[n]-0x80];
+    n--;
+  } else {
+err:
+    fprintf(stderr, "Error: unsupported opcode for %s symbol relocation.\n",
+      sym);
```

Comment:
Minor: indentation differs in [upstream](https://github.com/LuaJIT/LuaJIT/blob/v2.0/src/host/buildvm_asm.c#L70). Feel free to ignore.

Reply:
Sorry, but can't live with this mix of tabs and spaces. Ignoring.